### DISCUSSION TITLE
How are you guys and gals using SubtitleEdit in Linux?

### DISCUSSION THREAD

**Discussion by formlessdao** [SubtitleEdit/subtitleedit#6274] [Q&A]
What method of installing it on Linux offers the less bugs so far for you?

**Comment by niksedk** [comment]
Following this should work well: https://www.nikse.dk/subtitleedit/help#linux
Note that on latest Ubuntu/Fedora and probably others you need to set `MpvVideoOutputLinux` to `x11` in Settings.xml to make mpv work inside SE.
I've mostly tested SE on Ubuntu...

**Reply by felix2911-git** [reply]
This guide works well, with english subtitles. However, with german subtitles I have the issues that sometimes it just stops randomly at some subtitle. I have to pause there, extract the subtitle manually and restart at the next subtitle. Then it works again for a random number of subtitles. Are you aware of this issue?

And furthermore, the german dictionary on this way of running SubtilteEdit seems to be unaware of a lot of common german words. I do not observe this on windows with the same Version of SubtitleEdit.
